SuccessEycaActivationActivity/eyca.ts

```diff
@@ -0,0 +1,139 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { format } from "date-fns";
+import { toError } from "fp-ts/lib/Either";
+import {
+  fromEither,
+  fromLeft,
+  taskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import { RedisClient } from "redis";
+import { EycaAPIClient } from "../clients/eyca";
+import { Timestamp } from "../generated/definitions/Timestamp";
+import { CcdbNumber } from "../generated/eyca-api/CcdbNumber";
+import { ErrorResponse } from "../generated/eyca-api/ErrorResponse";
+import { errorsToError } from "../utils/conversions";
+import { getTask, setWithExpirationTask } from "../utils/redis_storage";
+
+export const CCDB_SESSION_ID_KEY = "CCDB_SESSION_ID";
+export const CCDB_SESSION_ID_TTL = 1500;
```

Comment:
I'd rather to maintain it as an hardcoded constant. It will never change. Therefore I will change the value to 1700 seconds avoiding that a minimum delay can compromise the token validity :)